actionview/lib/action_view/helpers/tags/password_field.rb

```diff
@@ -3,7 +3,7 @@ module Helpers
     module Tags # :nodoc:
       class PasswordField < TextField # :nodoc:
         def render
-          @options = {:value => nil}.merge!(@options)
+          @options = {:value => nil, :autocomplete => 'off' }.merge!(@options)
```

Comment:
It will be good to use new hash syntax here since Ruby 2.2 is needed for master now:

```ruby
@options = {value: nil, autocomplete: 'off' }.merge!(@options)
```